test/extended/cli/secret.go

```diff
@@ -0,0 +1,42 @@
+package cli
+
+import (
+	"path/filepath"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-cli] oc secret", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		oc       = exutil.NewCLI("oc-secret")
+		testData = exutil.FixturePath("testdata", "cmd", "test", "cmd", "testdata", "resource-builder")
+	)
+
+	g.It("creates and retrieves expected", func() {
+		g.By("creating secrets from a directory of files with proper extensions, as well as explicit filenames without extensions")
+		_, err := oc.Run("create").Args(
```

Comment:
Same here, whenever you don't care about output just use `Execute` instead of `Output`